features/support/remote_helpers.rb

```diff
@@ -26,6 +27,13 @@ def github_pull_request_url branch:, parent_branch: nil, repo:
   "https://github.com/#{repo}/compare/#{to_compare}?expand=1"
 end
 
+# Returns the URL for making merge requests on GitLab
+def gitlab_pull_request_url branch:, parent_branch:, repo:
+  parent_branch ||= 'main' # set default parent_branch to main
```

Comment:
I don't think this comment is needed. Also I believe this line can go away if you do the following a few lines down
```ruby
dest = "merge_request%5Btarget_branch%5D=#{parent_branch || 'main'}"
```